
0it [00:00, ?it/s]
09-10-21 13:52:39: Namespace(DeepJDOT_head=False, DefRec_dist='volume_based_voxels', DefRec_on_src=False, DefRec_on_trgt=True, DefRec_weight=0.5, alpha=0.6, apply_PCM=True, batch_size=16, dataroot='/scratch/shapenets/pointda', dropout=0.5, epochs=150, exp_name='DefRec_PCM', gpus=[0], lr=0.001, mixup_params=1.0, model='dgcnn', momentum=0.9, num_regions=3, optimizer='ADAM', out_path='./experiments', seed=1, softmax=False, src_dataset='shapenet', supervised=True, test_batch_size=32, trgt_dataset='scannet', use_DeepJDOT=False, wd=5e-05)
09-10-21 13:52:39: Using GPUs [0], from 1 devices available
09-10-21 13:52:39: number of train examples in shapenet: 17378
09-10-21 13:52:39: Occurrences count of classes in shapenet train set: {0: 599, 1: 167, 2: 310, 3: 1076, 4: 4612, 5: 1620, 6: 762, 7: 158, 8: 2198, 9: 5876}
09-10-21 13:52:45: number of train examples in scannet: 6110
09-10-21 13:52:45: Occurrences count of classes in scannet train set: {0: 98, 1: 329, 2: 464, 3: 650, 4: 2578, 5: 161, 6: 210, 7: 88, 8: 495, 9: 1037}
09-10-21 13:52:47: number of test examples in scannet: 1769
09-10-21 13:52:47: Occurrences count of classes in scannet test set: {0: 26, 1: 85, 2: 146, 3: 149, 4: 801, 5: 41, 6: 61, 7: 25, 8: 134, 9: 301}
09-10-21 13:52:47: Occurrences count of classes in source shapenet train part: {0: 480, 1: 134, 2: 248, 3: 860, 4: 3690, 5: 1296, 6: 610, 7: 126, 8: 1758, 9: 4702}
09-10-21 13:52:47: Occurrences count of classes in source shapenet validation part: {0: 119, 1: 33, 2: 62, 3: 216, 4: 922, 5: 324, 6: 152, 7: 32, 8: 440, 9: 1174}
09-10-21 13:52:47: Occurrences count of classes in target scannet train part: {0: 80, 1: 270, 2: 388, 3: 517, 4: 2070, 5: 130, 6: 164, 7: 71, 8: 391, 9: 807}





































































































































304it [04:28,  1.15it/s]
09-10-21 13:57:27: Trn - Source 0, total loss: 0.8046, mixup loss: 0.8046

305it [04:29,  1.13it/s]
09-10-21 13:57:49: Val - Source 0, acc: 0.7222, avg acc: 0.3249, cls loss: 0.8673
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
0it [00:00, ?it/s]





































































































































303it [04:26,  1.19it/s]
09-10-21 14:02:44: Trn - Source 1, total loss: 0.6290, mixup loss: 0.6290

305it [04:27,  1.14it/s]
09-10-21 14:03:06: Val - Source 1, acc: 0.8014, avg acc: 0.4739, cls loss: 0.6711
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 14:03:33: Val - Target 1, acc: 0.4468, avg acc: 0.3024, cls loss: 1.9081





































































































































304it [04:27,  1.15it/s]
09-10-21 14:08:02: Trn - Source 2, total loss: 0.5552, mixup loss: 0.5552

305it [04:28,  1.14it/s]
09-10-21 14:08:23: Val - Source 2, acc: 0.8382, avg acc: 0.5246, cls loss: 0.5523
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 14:08:51: Val - Target 2, acc: 0.3977, avg acc: 0.3061, cls loss: 2.1109






































































































































305it [04:29,  1.13it/s]
09-10-21 14:13:20: Trn - Source 3, total loss: 0.5247, mixup loss: 0.5247
09-10-21 14:13:20: Trn - Target 3, DefRec loss: 0.3428
09-10-21 14:13:41: Val - Source 3, acc: 0.8057, avg acc: 0.4832, cls loss: 0.6420
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 14:14:09: Val - Target 3, acc: 0.4173, avg acc: 0.2385, cls loss: 2.3193





































































































































305it [04:27,  1.14it/s]
09-10-21 14:18:36: Trn - Source 4, total loss: 0.5186, mixup loss: 0.5186
09-10-21 14:18:36: Trn - Target 4, DefRec loss: 0.3105
09-10-21 14:18:58: Val - Source 4, acc: 0.8374, avg acc: 0.5457, cls loss: 0.5851
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 14:19:25: Val - Target 4, acc: 0.4411, avg acc: 0.3343, cls loss: 2.1416





































































































































305it [04:27,  1.14it/s]
09-10-21 14:23:53: Trn - Source 5, total loss: 0.4971, mixup loss: 0.4971
09-10-21 14:23:53: Trn - Target 5, DefRec loss: 0.2934
09-10-21 14:24:14: Val - Source 5, acc: 0.8805, avg acc: 0.5769, cls loss: 0.4810
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 14:24:41: Val - Target 5, acc: 0.4534, avg acc: 0.3125, cls loss: 2.0024






































































































































305it [04:29,  1.13it/s]
09-10-21 14:29:11: Trn - Source 6, total loss: 0.4730, mixup loss: 0.4730
09-10-21 14:29:11: Trn - Target 6, DefRec loss: 0.2692
09-10-21 14:29:32: Val - Source 6, acc: 0.8566, avg acc: 0.5615, cls loss: 0.4890
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 14:29:59: Val - Target 6, acc: 0.4787, avg acc: 0.3452, cls loss: 2.0372




































































































































303it [04:25,  1.17it/s]
09-10-21 14:34:27: Trn - Source 7, total loss: 0.4715, mixup loss: 0.4715

305it [04:27,  1.14it/s]
09-10-21 14:34:48: Val - Source 7, acc: 0.8566, avg acc: 0.5601, cls loss: 0.5206
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 14:35:16: Val - Target 7, acc: 0.4337, avg acc: 0.2966, cls loss: 2.2540





































































































































303it [04:27,  1.20it/s]
09-10-21 14:39:45: Trn - Source 8, total loss: 0.4778, mixup loss: 0.4778

305it [04:28,  1.13it/s]
09-10-21 14:40:06: Val - Source 8, acc: 0.8175, avg acc: 0.5044, cls loss: 0.5371
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 14:40:33: Val - Target 8, acc: 0.4935, avg acc: 0.3131, cls loss: 1.9775





































































































































302it [04:25,  1.14it/s]
09-10-21 14:45:02: Trn - Source 9, total loss: 0.4569, mixup loss: 0.4569

305it [04:28,  1.14it/s]
09-10-21 14:45:23: Val - Source 9, acc: 0.8742, avg acc: 0.6488, cls loss: 0.4657
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 14:45:50: Val - Target 9, acc: 0.4730, avg acc: 0.3039, cls loss: 1.9917






































































































































304it [04:29,  1.12it/s]
09-10-21 14:50:21: Trn - Source 10, total loss: 0.4489, mixup loss: 0.4489

305it [04:30,  1.13it/s]
09-10-21 14:50:42: Val - Source 10, acc: 0.8923, avg acc: 0.6351, cls loss: 0.3986
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 14:51:09: Val - Target 10, acc: 0.4935, avg acc: 0.3903, cls loss: 1.9885





































































































































303it [04:27,  1.15it/s]
09-10-21 14:55:39: Trn - Source 11, total loss: 0.4569, mixup loss: 0.4569

305it [04:28,  1.13it/s]
09-10-21 14:56:00: Val - Source 11, acc: 0.8849, avg acc: 0.6376, cls loss: 0.4480
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 14:56:27: Val - Target 11, acc: 0.4689, avg acc: 0.3448, cls loss: 2.0623




































































































































304it [04:25,  1.16it/s]
09-10-21 15:00:54: Trn - Source 12, total loss: 0.4585, mixup loss: 0.4585

305it [04:26,  1.14it/s]
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 15:01:15: Val - Source 12, acc: 0.8788, avg acc: 0.6129, cls loss: 0.4325
09-10-21 15:01:42: Val - Target 12, acc: 0.4681, avg acc: 0.3177, cls loss: 2.1780





































































































































303it [04:26,  1.12it/s]
09-10-21 15:06:11: Trn - Source 13, total loss: 0.4333, mixup loss: 0.4333

305it [04:28,  1.14it/s]
09-10-21 15:06:32: Val - Source 13, acc: 0.8909, avg acc: 0.6447, cls loss: 0.4092
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x1533cd7bf830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 15:06:59: Val - Target 13, acc: 0.4427, avg acc: 0.2922, cls loss: 2.2165

























































































































